content/en/docs/getting-started/_index.md

```diff
@@ -60,73 +61,141 @@ olm-operator       1/1     1            1           5m52s
+## Installing OLM and operator bundle via SDK for development purpose
 
-```sh
-$ kubectl get packagemanifest -n olm
-NAME                               CATALOG               AGE
-cassandra-operator                 Community Operators   26m
-etcd                               Community Operators   26m
-postgres-operator                  Community Operators   26m
-prometheus                         Community Operators   26m
-wildfly                            Community Operators   26m
+### Installing OLM
+
+The [`operator-sdk`][operator-sdk] provides a command to easily install and uninstall OLM for development purposes. See the [SDK installation guide][sdk-installation-guide] on how to install `operator-sdk`.  
+
+With `operator-sdk` installed, you can easily install OLM on your cluster by running `operator-sdk olm install`. It is just as easy to uninstall OLM by running `operator-sdk uninstall olm`.  For more, information about how to integrate your project using [`operator-sdk`][operator-sdk] CLI tool see the [OLM integration][sdk-olm-integration] section. 
+
+### Installing an operator bundle
+
+You can use the [`operator-sdk`][operator-sdk] CLI to run your bundle with [`operator-sdk run bundle`](https://sdk.operatorframework.io/docs/cli/operator-sdk_run_bundle/).
+
+## Running OLM locally with minikube for development purpose
+
+This command starts minikube, builds the OLM containers locally with the minikube-provided docker, and uses the local configuration in [local-values.yaml][local-values.yaml] to build localized deployment resources for OLM.
+
+```bash
+# To install and run locally
+$ make run-local
 ```
 
-To install the etcd operator in the default namespace, first create an `OperatorGroup` for the default namespace:
+You can verify that the OLM components have been successfully deployed by running `kubectl -n local get deployments`
 
-```sh
-$ cat operatorgroup.yaml
-kind: OperatorGroup
-apiVersion: operators.coreos.com/v1
-metadata:
-  name: og-single
-  namespace: default
-spec:
-  targetNamespaces:
-  - default
+## User Interface (Running the console Locally)
+
+Use the admin console (compatible with upstream Kubernetes) to interact with and visualize the resources managed by OLM. Create subscriptions, approve install plans, identify Operator-managed resources, and more.
+
+Ensure `kubectl` is pointing at a cluster and run:
 
-$ kubectl apply -f operatorgroup.yaml
-operatorgroup.operators.coreos.com/og-single created
+```shell
+$ make run-console-local
 ```
 
-Then create a subscription for the etcd operator:
+Then visit `http://localhost:9000` to view the console.
+
+## Customizing OLM installation 
+
+Deployments of OLM can be stamped out with different configurations by writing a `values.yaml` file and running commands to generate resources.
+
+Here's an example `values.yaml`
+
+```yaml
+# sets the apiversion to use for rbac-resources. Change to `authorization.openshift.io` for openshift
```

Comment:
Just for consistency-sake.

```suggestion
# sets the apiversion to use for rbac-resources. Change to `authorization.openshift.io` for openshift.
```

Reply:
Well, nevermind :laughing: - I don't have a strong opinion, but it doesn't look like there's any consistency for the comments in this yaml code block, so disregard if you want.